template.go

```diff
@@ -80,3 +80,14 @@ func (c *Context) MatchCondition(condition string) (r bool) {
 	}
 	return result == "true"
 }
+
+// MatchConditions checks the AND of all conditions using MatchCondition
+func (c *Context) MatchConditions(conditions []string) (r bool) {
+	result := true
+
+	for _, condition := range conditions {
+		result = result && c.MatchCondition(condition)
```

Comment:
nit: you can probably short-circuit this with `if !c.MatchCondition(condition) { return false }`

Reply:
I initially decided not to return early because of the logging that is done in `MatchCondition`, but I can change this